### ISSUE TITLE
size_frac_up

### ISSUE THREAD

**Issue by kmexter** [emo-bon/observatory-vb-crate#6]
The water logsheet has 9999 in the side_frac_up and I think they mean NA. I have added a comment to the logsheet, but can @melanthia and @melinalou take care of ensuring that they answer the comment? 
https://docs.google.com/spreadsheets/d/1kSlTcNfXaCQv7fIKVbBnt5PmRW3_H7Trryh4FJHyLkE/edit?gid=753216632#gid=753216632

of course, is it really is 9999, then it can stay like that :-D

**Comment by melinalou** [comment]
email sent

**Comment by melinalou** [comment]
![image](https://github.com/user-attachments/assets/52503df2-3ede-4072-b16f-1c4874aa1892)

**Comment by kmexter** [comment]
Yup, where there is a lower limit but no upper limit you put NA in the upper limit box -> tell them that, but you can make the corrections yourself

**Comment by melinalou** [comment]
good!

**Comment by melinalou** [comment]
done, issue can be closed
